templates/zrobot_config/zrobot_config.py

```diff
@@ -28,12 +29,31 @@ def install(self):
         install writes the config, then send a signal to the robot to stop itself
         in production 0-os will detect that the robot stops and will restart it
         """
-
+        if not j.sal.fs.exists(CONFIG_PATH):
```

Comment:
this is not needed, the next line will also create the file `j.data.serializer.yaml.dump(...)`